lib/src/Base/Func/NormInfEnumerateFunction.cxx

```diff
@@ -143,6 +144,22 @@ UnsignedInteger NormInfEnumerateFunction::getMaximumDegreeStrataIndex(const Unsi
   return maximumDegree / getDimension();
 }
 
+/* The marginal enumerate function */
+EnumerateFunction NormInfEnumerateFunction::getMarginal(const Indices & indices) const
+{
+  const UnsignedInteger inputDimension = getDimension();
+  indices.check(inputDimension);
+  const UnsignedInteger activeDimension = indices.getSize();
+  const NormInfEnumerateFunction enumerateFunctionMarginal(activeDimension);
+  return enumerateFunctionMarginal;
+}
+
+/* The marginal enumerate function */
+EnumerateFunction NormInfEnumerateFunction::getMarginal(const UnsignedInteger i) const
+{
+  return getMarginal(Indices({i}));
+}
```

Comment:
here too

Reply:
Fixed at d17f1462144022b009756e26e0d7de62313a3207

Reply:
Oups:
```
151/958 Test #1073: pyinstallcheck_NormInfEnumerateFunction_std ......................................***Failed    0.29 sec
Traceback (most recent call last):
  File "/home/devel/project/python/test/t_NormInfEnumerateFunction_std.py", line 43, in <module>
    marginalf = f.getMarginal(3)
                ^^^^^^^^^^^^^^^^
  File "/home/devel/.local/lib/python3.12/site-packages/openturns/func.py", line 2009, in getMarginal
    return _func.NormInfEnumerateFunction_getMarginal(self, indices)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Object passed as argument is not convertible to an Indices
```

Reply:
Oups: it fails.

Reply:
Ok. Fixed, again.